src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,14 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        Ball firstBall = new Ball();
+        Ball secondBall = new Ball();
+        Ball thirdBall = new Ball();
+        Ball[] balls = new Ball[] {firstBall, secondBall, thirdBall};
+        for (Ball ball : balls) {
+            ball = lottery.getRandomBall();
```

Comment:
The assignment `ball = lottery.getRandomBall();` only updates the local variable `ball` within the loop, not the elements in the `balls` array. To store the random `Ball` objects in the array, you should assign the result of `lottery.getRandomBall()` back to the array, e.g., `balls[i] = lottery.getRandomBall();` where `i` is the index of the loop.